Makefile

```diff
@@ -7,11 +7,13 @@ test :
 
 # install dropboxignore
 install :
+	sudo apt install -y attr
```

Comment:
The purpose of dropbox ignore is be as cross-platform as possible. `apt` is the package manager for debian and debian-based systems. As a result, this `Makefile`could not be used by other platforms.
```suggestion
```